pkg/chunk/table_manager.go

```diff
@@ -37,11 +43,29 @@ var (
 		Name:      "dynamo_table_capacity_units",
 		Help:      "Per-table DynamoDB capacity, measured in DynamoDB capacity units.",
 	}, []string{"op", "table"})
+	retentionPeriodConfig = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "cortex",
+		Name:      "retention_period",
+		Help:      "Configured retention period",
+	})
+	tableNumber = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
What do you buy from having the `extreme` and `kind` labels? The Prometheus rule of thumb is that it should be meaningful to aggregate across all labels, and I am not convinced of that right now.

Reply:
I am going to remove extreme label. Do you think there should be different guages for chunk and index table instead of having a single guage with `kind` label?